### PR TITLE
Build ui

### PR DESCRIPTION
**Pull request by xiaoshi22**
```
----------------------------------------------------------------------
XML: /vagrant/unittests.xml
Name                  Stmts   Miss  Cover   Missing
---------------------------------------------------
service/__init__.py      18      0   100%
service/models.py       106      0   100%
service/service.py      205      1    99%   379
---------------------------------------------------
TOTAL                   329      1    99%
----------------------------------------------------------------------
Ran 89 tests in 9.752s

OK
```
[](url)

### CODE REVIEW

File: service/service.py [hunk 1]
```diff
@@ -78,13 +78,14 @@ def internal_server_error(error):
...
     """ Root URL response """
     app.logger.info("Request for Root URL")
-    return (
-        jsonify(
-            name="Orders REST API Service",
-            version="1.0"
-        ),
-        status.HTTP_200_OK,
-    )
+    # return (
```
Comment (shivkhattar) [inline service/service.py:81]: We don't need to keep this commented. You can remove this completely.

File: tests/test_service.py [hunk 2]
```diff
@@ -82,9 +82,10 @@ def test_index(self):
         """ Home Page """
         resp = self.app.get("/")
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
-        data = resp.get_json()
-        self.assertEqual(data["name"], "Orders REST API Service")
-        self.assertEqual(data["version"], "1.0")
+        # data = resp.get_json()
+        # self.assertEqual(data["name"], "Orders REST API Service")
```
Comment (shivkhattar) [inline tests/test_service.py:86]: Same here. Remove the comment.

**Review by shivkhattar — APPROVED** [review]

**Review by YufeiGao6 — APPROVED** [review]

**Comment by shivkhattar** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/d97e2552-e5d6-4314-93cd-4603445a8066?env_id=ibm:yp:us-south "orders-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/de2c1c4b-d454-4678-8c83-87825f7d8ab3 "order-pipeline") deployed [orders](https://github.com/NYU-Devops-2020-Orders-Team/orders/releases/tag/deploy-ibm-yp-us-south-sk8325-nyu.edu-dev-20201110-230329 "deploy-ibm-yp-us-south-sk8325-nyu.edu-dev-20201110-230329") to [dev](https://console.bluemix.net/apps/2df1ce90-4dbe-4be7-ae24-bb8fa0a1537e?env_id=ibm:yp:us-south "ibm:yp:us-south:sk8325@nyu.edu:dev"), including commit 47553fa72bcc6f0d92cafc2e3fb34a9ecb3aad55

**Comment by shivkhattar** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/d97e2552-e5d6-4314-93cd-4603445a8066?env_id=ibm:yp:us-south "orders-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/de2c1c4b-d454-4678-8c83-87825f7d8ab3 "order-pipeline") deployed [orders](https://github.com/NYU-Devops-2020-Orders-Team/orders/releases/tag/deploy-ibm-yp-us-south-sk8325-nyu.edu-prod-20201116-013849 "deploy-ibm-yp-us-south-sk8325-nyu.edu-prod-20201116-013849") to [prod](https://console.bluemix.net/apps/72e50e75-2807-435f-b5d1-e24358ccc3c9?env_id=ibm:yp:us-south "ibm:yp:us-south:sk8325@nyu.edu:prod"), including commit 47553fa72bcc6f0d92cafc2e3fb34a9ecb3aad55

**Pull request merged**